react/tic-tac-toe/src/app/screens/Login/validation/index.js

```diff
@@ -0,0 +1,12 @@
+export const required = value =>
```

Comment:
This validations could be used by many other components in the app. (May be not in the training but sure they do on real life projects).
So, i'w move them into a top level `utils` directory.
`src/app/utils/inputValidations.js`